### DISCUSSION TITLE
Solr localhost

### DISCUSSION THREAD

**Discussion by lottehaupt** [irgroup-classrooms/dis17-2021#11] [Q&A]
Hi,
habe meine Solr Anwendung in der Git Shell gestartet, kann sie aber nicht im localhost öffnen. Hat jemand eine Idee woran das liegen könnte?

![image](https://user-images.githubusercontent.com/63407473/140536210-f108c89a-b5ab-4067-ae74-1b82952971e2.png)
 
Danke

**Comment by MehmetCagferoglu** [comment]
Hey, was hast du denn für ein Betriebssystem? Falls du Windows hast, mach das am besten über Start und dann "cmd" so konnte ich ohne Probleme meine beiden  localhost starten. 
Hier noch ein Link wie du Solr und Java richtig installierst.
Sag Bescheid falls noch Probleme entstehen sollten! 

https://www.youtube.com/watch?v=_fW98B84lXA

**Reply by xxjuly** [reply]
Kann sein, dass du wegen deinem Internetanbieter (Vodafone spinnt da was rum) und die dazugehörige Firewall nicht weiter kommst. Versuche mal den DNS Server zu ändern. Hier eine genaue Erklärung dazu: https://praxistipps.chip.de/windows-10-dns-server-aendern-so-gehts_47055

**Reply by lottehaupt** [reply]
Hat funktioniert, danke! @MehmetCagferoglu
